modules/studio-service/src/main/java/org/opencastproject/studio/endpoint/dto/SeriesDto.java

```diff
@@ -0,0 +1,43 @@
+/**
+ * Licensed to The Apereo Foundation under one or more contributor license
+ * agreements. See the NOTICE file distributed with this work for additional
+ * information regarding copyright ownership.
+ *
+ *
+ * The Apereo Foundation licenses this file to you under the Educational
+ * Community License, Version 2.0 (the "License"); you may not use this file
+ * except in compliance with the License. You may obtain a copy of the License
+ * at:
+ *
+ *   http://opensource.org/licenses/ecl2.txt
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.  See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ *
+ */
+
+package org.opencastproject.studio.endpoint.dto;
+
+public class SeriesDto {
+
+  private String id;
+
+  private String title;
+
+  public SeriesDto(String id, String title) {
+    this.id = id;
+    this.title = title;
+  }
+
+  public String getId() {
+    return id;
+  }
+
+  public String getTitle() {
+    return title;
+  }
+
+}
```

Comment:
It would be great, if `SeriesDto` wouldn't need to live in its own file and if it didn't need a constructor and getters. Unfortunately, public fields are currently rejected by the style checker, but this is something I wanted to get changed for a long time already. Maybe I'll finally get to bringing this up in the community.

Reply:
Java has the record keyword for that since Java 14